README.md

```diff
@@ -239,6 +239,141 @@ action := NewTwoStateToolBar(theme.MediaPlayIcon(),
 
 * [Demo App](cmd/twostatetoolbaraction_demo/main.go)
 
+### NumericalEntry
```

Comment:
This is over 130 lines of text for a single widget - in the README that covers the whole repo.
Can it be simplified - or if required to be this in-depth some of it expand into another file?

Reply:
Most of the description of NumericalEntry has been moved to a separate README_numericalentry.md file.